dashboard/app/views/levels/_open_response.html.haml

```diff
@@ -0,0 +1,16 @@
+:javascript
+  function getResult() {
+    return {
+      response: $('.response').val(),
+      result: true
+    };
+  }
+
+#open-response
+  %div= ActionView::Base.new.render(inline: @level.markdown_instructions, type: :md)
+
+  - height = @level.height || '80'
+  %textarea.response{placeholder: @level.placeholder, style: "width: 700px; height: #{height}px;", readonly: @view_options.readonly_workspace}= @last_attempt
+
+= render partial: 'levels/dialog', locals: {app: 'open_response', data: @level.properties}
+= render partial: 'levels/teacher_markdown', locals: {data: {'teacher_markdown' => @level.solution}}
```

Comment:
the css styles for this are nested in #match/#multi/whatever so you will have to add #open-response here: 

https://github.com/code-dot-org/code-dot-org/blob/e6a37edd9e2fb2fad4b45c0ebd8671580d5b0fac/dashboard/app/assets/stylesheets/application.css.scss#L1634